### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Add a resource via Azure Active Directory_Attach with Azure AD _ScreenReader: Instruction is not provided prior to entering url on what the error is how to solve it r that the field is required 

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2095]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS3.3.2;#NVDA;#JAWS; #Win32; 

Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
NVDA 2019.2
JAWS 2019
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with azure AD" screen gets opened and navigate to Next button and hit enter.
5.Open NVDA and check whether it reads Instructions prior to entering url on what the error is how to solve it r that the field is required .
**​Actual:​**
Instruction is not provided prior to entering url on what the error is how to solve it r that the field is required 
Same issue is observed in JAWS also.
**Expected:**
Instruction should be provided prior to entering url on what the error is how to solve it r that the field is required 
UserImpact:
Screen reader users unable to get that what type of URL should be put.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo , can you please elaborate on what is wrong? The instructions for the page are read by the screen reader, and if you enter something "wrong" in the field, then the error is alerted....

Here's what NVDA says:
Microsoft Azure Storage Explorer - Connect
Attach with Azure AD  dialog
Choose the resource type you want to attach, and then enter the information needed to connect.
Resource type:  combo box  Blob Container  collapsed
Container URL:  edit
blank
t
alert  No protocol specified in the URL. Specify either HTTP or HTTPS.

**Comment by MRayermannMSFT** [comment]
If I don't hear a reply within 3 days, I'll go ahead and close the bug.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
Earlier Screen reader not announces the alert message "Connecting over http is not recommended. Consider using https instead".
Again Verified this issue with application version 1.10.0 now screen reader announces the alert message, and this issue is no longer reproduced.
Here screen reader works fine and announces the alert message.

**Comment by MRayermannMSFT** [comment]
Ok, closing the bug then. Thank you @suraj1200499 .

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and the issue is no longer reproduced.Hence,close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
